cmd/meroxa/flink/deploy.go

```diff
@@ -24,77 +26,105 @@ const (
 	irVal            = "EMIT_IR"
 )
 
+func doesJobUseMeroxaPlatform(ctx context.Context, jarPath string) (bool, error) {
```

Comment:
I think this approach is fine for now - do you want to leave a TODO comment (with a github issue) indicating that we need to verify that the `execute()` method is called on `MeroxaExecutionEnvironment`?